src/main/resources/assets/onebar/lang/zh_cn.json

```
@@ -0,0 +1,131 @@
+{
+  "modmenu.descriptionTranslation.onebar": "只在你需要时显示长条和数值的界面模组",
+  "text.onebar.health": "命%s",
+  "text.onebar.absorption": "吸%s",
+  "text.onebar.hunger": "饿%s",
+  "text.onebar.resistance": "抗%s%%",
+  "text.onebar.invisibility": "隐",
+  "text.onebar.air": "气%s",
+  "text.onebar.freeze": "冷%s",
+  "text.onebar.fire": "火%s×",
+  "text.onebar.warden": "监%s",
+  "text.onebar.hardcore": "HC",
+  "text.onebar.badOmen": "咒%sx",
+  "text.onebar.mobHeadSkeleton": "骷",
+  "text.onebar.mobHeadCreeper": "苦",
+  "text.onebar.mobHeadZombie": "僵",
+  "text.onebar.mobHeadEnderman": "黑",
+  "text.onebar.mobHeadPiglin": "猪",
+  "text.onebar.healthEmoji": "\u2764%s",
+  "text.onebar.absorptionEmoji": "\u2665%s",
+  "text.onebar.hungerEmoji": "\uD83C\uDF56%s",
+  "text.onebar.resistanceEmoji": "\u26E8%s%%",
+  "text.onebar.invisibilityEmoji": "\u2664",
+  "text.onebar.airEmoji": "\u2B58%s",
+  "text.onebar.freezeEmoji": "\u2744%s",
+  "text.onebar.fireEmoji": "\uD83D\uDD25%s×",
+  "text.onebar.wardenEmoji": "§l\u274C§r%s",
+  "text.onebar.hardcoreEmoji": "\u2620",
+  "text.onebar.badOmenEmoji": "\ud83e\ude93%sx",
+  "text.onebar.mobHeadSkeletonEmoji": "s\u263B",
```

```suggestion
  "text.onebar.mobHeadSkeletonEmoji": "骷\u263B",
```
First letter represents the mob. Do the same for other strings below